core/tests/protractor_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -164,7 +168,10 @@ var ExplorationEditorFeedbackTab = function() {
       by.css('.protractor-test-oppia-feedback-status-name'));
     await waitFor.visibilityOf(
       feedbackStatusElement, 'Feedback status is not visible.');
-    expect(await feedbackStatusElement.getText()).toEqual(feedbackStatus);
+    await waitFor.textToBePresentInElement(
+      feedbackStatusElement,
+      feedbackStatus,
+      `Expected ${feedbackStatusElement.getText()} to be ${feedbackStatus}`);
```

Comment:
does this work? `getText`  returns a [promise](https://www.protractortest.org/#/api?view=webdriver.WebElement.prototype.getText), you will need to use await.

Reply:
Done